xtable-utilities/pom.xml

```diff
@@ -36,6 +36,24 @@
         <dependency>
             <groupId>org.apache.xtable</groupId>
             <artifactId>xtable-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>javax.servlet</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.servlet.jsp</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.eclipse.jetty</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.antlr</groupId>
+                    <artifactId>antlr4-runtime</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Excluding servlet and antlr-runtime dependencies, since they're not being used in utilities.

Reply:
servlet jars are coming from multiple dependencies like hudi-common, and hbase-server as shown below. Antlr is coming from delta-core

[INFO] +- org.apache.xtable:xtable-core:jar:0.2.0-SNAPSHOT:compile
[INFO] |  +- org.apache.hudi:hudi-java-client:jar:0.14.0:compile
[INFO] |  |  \- org.apache.hudi:hudi-client-common:jar:0.14.0:compile
[INFO] |  |     +- org.apache.hudi:hudi-timeline-service:jar:0.14.0:compile
[INFO] |  |     |  +- io.javalin:javalin:jar:4.6.7:compile
[INFO] |  |     |  |  +- org.eclipse.jetty:jetty-server:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  +- org.eclipse.jetty:jetty-http:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  \- org.eclipse.jetty:jetty-io:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  +- org.eclipse.jetty:jetty-webapp:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  +- org.eclipse.jetty:jetty-xml:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  \- org.eclipse.jetty:jetty-servlet:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |     \- org.eclipse.jetty:jetty-security:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  +- org.eclipse.jetty.websocket:websocket-server:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  +- org.eclipse.jetty.websocket:websocket-common:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  |  \- org.eclipse.jetty.websocket:websocket-api:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  +- org.eclipse.jetty.websocket:websocket-client:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  |  \- org.eclipse.jetty:jetty-client:jar:9.4.54.v20240208:compile
[INFO] |  |     |  |  |  \- org.eclipse.jetty.websocket:websocket-servlet:jar:9.4.54.v20240208:compile

] +- org.apache.xtable:xtable-core:jar:0.2.0-SNAPSHOT:compile
[INFO] |  +- com.fasterxml.jackson.module:jackson-module-scala_2.12:jar:2.17.1:compile
[INFO] |  +- org.apache.hudi:hudi-common:jar:0.14.0:compile
[INFO] |  |  +- org.apache.hbase:hbase-server:jar:2.4.9:compile
[INFO] |  |  |  +- javax.servlet.jsp:javax.servlet.jsp-api:jar:2.3.1:compile

[INFO] +- org.apache.xtable:xtable-core:jar:0.2.0-SNAPSHOT:compile
[INFO] |  +- org.apache.xtable:xtable-hudi-support-utils:jar:0.2.0-SNAPSHOT:compile
[INFO] |  \- io.delta:delta-core_2.12:jar:2.4.0:compile
[INFO] |     +- io.delta:delta-storage:jar:2.4.0:compile
[INFO] |     \- org.antlr:antlr4-runtime:jar:4.9.3:compile

Reply:
I see below dependencies are present in the current dependency tree for utilties module. Will try to remove them

jersey-json
jersey-core
jersey-server
jersey-servlet
jaxb-impl
javax.activation-api
javax.annotation-api
javax.servlet.jsp-api
javax.servlet-api
javax.ws.rs-api
jsr311-api
jaxb-api
javax.servlet.jsp
javax.el
jol-core

Reply:
Xtable sync is failing on removing dependencies 

jersey-server
jol-core